src/components/EditableLabel/EditableLabel.tsx

```diff
@@ -111,6 +114,16 @@ const EditableLabel = ({
 
   useDetectOutsideClick(containerRef, handeOutsideClick)
 
+  const shouldTooltipBeDisabled: () => boolean = () => {
```

Comment:
proponuję zdefiniować typ u góry pliku, a tutaj go użyć; w pierwszej chwili wprowadza lekki zamęt w głowie :p

Reply:
jakaś propozycja na nazwę typu? Coś w stylu "funkcja zwracająca boolean" trochę dziwne xd

Reply:
nazwałabym coś w rodzaju booleanFuncType, albo jeżeli ts dobrze się tutaj domyśla to nie typowała

Reply:
to nie typuję, TS ogarnia ;p Chciałem być explicit ale w sumie nie trzeba.